test/components/global-event.spec.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import {render, unmountComponentAtNode} from 'react-dom';
```

Comment:
Lets keep tests with `test-drive-react` to render instead of directly using `react-dom`.

If there is a problem with un-mounting during the test, then talk to @AviVahl about adding the option in the render result.